module/vision/BallDetector/src/BallDetector.hpp

```diff
@@ -27,6 +27,7 @@ namespace module::vision {
 
     class BallDetector : public NUClear::Reactor {
     private:
+        // See BallDetector.yaml for info on configuration variables and their values
```

Comment:
If we are using doxygen-like documentation system you should duplicate the definitions of these config quantities here so that they show up in the docs (unless doxygen can parse yaml files???)